invenio_vocabularies/datastreams/writers.py

```diff
@@ -79,20 +84,35 @@ def _entry_id(self, entry):
     def _resolve(self, id_):
         return self._service.read(self._identity, id_)
 
+    def _do_update(self, entry):
+        vocab_id = self._entry_id(entry)
+        current = self._resolve(vocab_id)
+
+        updated = dict(current.to_dict(), **entry)
+        # TODO: Try to use _record instead of to_dict()
```

Comment:
nit: what to do with this `TODO`?

Reply:
These two commented-out lines can be removed.
Alex suggested to try `_record` instead of `to_dict()` but I don't think it matters.